packages/website/src/components/Announcement.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import Text from './Text';
+import { H4 } from './Heading';
+import Box from './Box';
+import Section from './Section';
+
+interface AnnouncementProps {
+  location: Object;
+}
+
+const AnnouncementBox = styled(Box)`
+  background: #ffe9bf;
+  width: 60%;
+  padding: ${({ theme }) => theme.space.medium} ${({ theme }) => theme.space.large};
+  margin-bottom: ${({ theme }) => theme.space.medium};
+  border-radius: 4px;
+`;
+
+const redirected = (props): boolean => {
+  const params = new URLSearchParams(props.search);
+  const from_old_website: string = params.get('from-old-website')!;
+  return from_old_website == 'true' ? true : false;
+};
+
+const from_where = props => {
+  const params = new URLSearchParams(props.search);
+  const from_which_page: string = params.get('archived-page')!;
+  return (
+    <Text>
+      You can still find the page you were trying to visit in our archive:{' '}
+      <a href={from_which_page}>{from_which_page}</a>
+    </Text>
+  );
+};
+
+const Announcement: React.FC<AnnouncementProps> = props => {
+  if (redirected(props.location)) {
```

Comment:
I'm not convinced by this pattern. I'd think it's more intuitive if the parent determines whether this component is visible, rather than the component determining itself whether to show up or not. But perhaps I'm way too used to thinking in ember terms? cc. @taras 

If we're to leave this logic inside the component, we'd have to rename this component to something more specific, as `Announcement` reads more like a general-use piece.

Reply:
I agree that we should probably rename this component to `<MaybeGreetLegacySiteUsers>`

Reply:
oh yeah, i put the logic in index at first but moved it into components to make it look nicer. so should i rename the component to `<Maybe...>` or move the logic onto the index page? @cowboyd

Reply:
Try both and ask yourself which one you like best and why?

Reply:
Did some refactoring to simplify the logic and renamed the component to be more explicit. 

I like the idea of keeping components modular so for now I decided to keep the conditional inside the component as this logic only applies for this component only at the moment.